addons/web/static/src/core/tree_editor/tree_editor.xml

```diff
@@ -145,11 +145,14 @@
     </t>
 
     <t t-name="web.TreeEditor.condition:readonly">
-        <t t-set="description" t-value="getDescription(node)" />
+        <t t-set="description" t-value="leafToDescription(node)" />
         <div class="o_tree_editor_condition d-flex gap-1 px-2 border bg-100">
-            <ModelFieldSelector path="node.path" resModel="props.resModel" readonly="true" />
-            <span class="fst-italic" t-esc="description.operatorDescription" />
-            <t t-if="description.valueDescription">
+			<div class="fw-bolder text-nowrap" t-esc="description.pathDescription" />
```

Comment:
not the same html as before (see model_field_selector.xml)
--> also, shouldn't we use the path editor from getPathEditorInfo in readonly?